src/tests/userSpec.js

```diff
@@ -3,8 +3,8 @@ import chaiHttp from 'chai-http';
 import app from '..';
 
 chai.use(chaiHttp);
-const { expect } = chai;
-
+const { expect, assert, should } = chai;
```

Comment:
Any reason why you added assert and should here, is there anything you were unable to achieve with `expect` here

Reply:
I honestly didn't think 'expect' could do most. 
thanks